### PR TITLE
Revert "Add hooks for :target pseudo-selector"

### PR DESCRIPTION
**Pull request by bholley**
This reverts commit 11b01919b8e160c5d93c0bb496fea5062307f0dc, which breaks servo.

[Review on Reviewable](https://reviewable.io/reviews/servo/rust-selectors/54)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "selectors"
-version = "0.1.1"
+version = "0.1.0"
```
Comment (jdm) [inline Cargo.toml]: We should increase this, instead.

**Comment by SimonSapin** [comment]
Hang on, why is it justified to revert this, exactly? The only reason I can imagine this breaking Servo is that Servo doesn’t implement `get_target_state`. The solution is not to revert things, it’s to implement `get_target_state`, even if it is (for now) a dummy implementation that always returns `false`.

**Comment by bholley** [comment]
@SimonSapin I tried doing that, but @Ms2ger and @jdm objected. See https://github.com/servo/servo/pull/8142#issuecomment-150151752.

**Pull request merged**